content/uiux/concepts/fitts-law/fitts-law.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: 'Fitts Law'
+Description: 'Add term entry for Fitts Law to UI and UX Design glossary'
+Subjects:
+  - 'Developer Tools'
+  - 'Mobile Development'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Design'
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Fitts' Law**, initially developed by Paul Fitts, is a term related to UI and UX Design describing how movement time to a target (link, button, etc.), with a pointer (mouse, finger, etc.), is a function of the distance to the target as well as the size of the target. Where poor functionality was once attributed solely to human error, Fitts first recognized this was also due to poor design.
+
+Fitts determined that larger targets with minimal spacing allow for better usability in design. However, caution should be exercised when limiting spacing, as clutter can lead to poor functionality. When designing a UI, it is crucial to recognize the user's workflow on a given page and strategically place targets based on the predicted pointer placement. The larger a target, the better it is for usability and also reduces user errors. Clustering commonly used items and menus should be considered for optimal UX.
+
+## Example
+
+The gif below depicts a basic demonstration of Fitts' Law. The varying size and placement of targets illustrate the differing levels of user accuracy in acquiring each target as well as the speed of acquiring each target.
+
+![Fitts' Law demonstration](https://raw.githubusercontent.com/Codecademy/docs/main/media/fitts-law.gif)
+
+Notice how smaller targets with large spacing decreased the accuracy and speed of acquiring each target, while larger targets with minimal spacing increased accuracy and speed. Despite the highest accuracy and speed shown on the largest and overlapping target set, this would not be considered good design practice, as the targets are cluttered and could lead to poor usability. The goal is to find the optimal size and spacing so that targets are presented clearly, and are easily distinguishable from one another.
```

Comment:
```suggestion
Notice how smaller targets with large spacing decreased the accuracy and speed of acquiring each target, while larger targets with minimal spacing increased accuracy and speed. Despite the largest and overlapping target set showing the highest accuracy and speed, this would not be considered good design practice, as the targets are cluttered and could lead to poor usability. The goal is to find the optimal size and spacing so that targets are easily distinguishable from one another.
```